Resilio Connect API/JSMonitor/app.js

```diff
@@ -28,9 +30,9 @@ setTimeout(function(){console.log (getAgentProperty(1, "status"));}, 3000);
 
 updateJobsPerAgent();
 
-setTimeout(function(){console.log (getJobProperty(2, "agents"));}, 3000)
-
-
+setTimeout(function(){console.log (getJobProperty(2, "agents"));}, 3000);
 
+setTimeout(function() {console.log(enumerateAgents());}, 5000);
 
+setTimeout(function() {console.log(findArrayDiff([1, 2, 3, 4], [1, 2]));}, 6000)
```

Comment:
No reason to this with a timeout.  The others needed an arbitrary timeout as an easy workaround for the async nature of previous calls.  findArrayDiff doesn't depend on anything async.